File: C++/Number_of_digits.cpp
#include<iostream>
using namespace std;
int main(){
    int n,temp;
    temp=0;
    cout<<"Enter number : ";
    cin>>n;
    while(n!=0){
        n=n/10;
        temp++;
    }
    cout<<temp;
    return 0;
}{

File: Java/Number_of_digits.java
package com.lev;

import java.util.Scanner;

public class Number_of_digits {
    public static void main(String[] args) {
        System.out.print("Enter number : ");
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int temp = 0;
        while(n != 0){
            n = n / 10;
            temp++;
        }
        System.out.print(temp);
    }
}
